### ISSUE TITLE
Error when you delete a box but move to another page before it completes

### ISSUE THREAD

**Issue by YaManicKill** [porybox/porybox#693]
I haven't investigated this, but I just noticed that if you delete a box, and then move to another page (say, the index) you get a huge error in the console. It doesn't seem to have any other side-effects, and I doubt people are doing this much, so it's probably not a high priority.

**Comment by pkmnbumba** [comment]
Adding what I noticed from dupe issue: 
If you press on Porygon to return to the main page after deleting a box (but before the "Box is deleted" message), the deleted box doesn't disappear until you refresh. (only occurs if you move quickly enough)

I haven't gotten the chance to look at the design of porybox yet so I'm not too sure how it works but possible fix would be to refresh the UI after it says "Box is deleted"

**Comment by YaManicKill** [comment]
Nah, it'll be a simpler fix than that. I've not looked into exactly what the issue is, but I reckon when you move the page, some reference to some object changes, which is what we are trying to remove from on the box deletion.
